api/envoy/config/filter/http/jwt_authn/v2alpha/BUILD

```diff
@@ -11,3 +11,13 @@ api_proto_library_internal(
         "//envoy/api/v2/route",
     ],
 )
+
+api_go_proto_library(
```

Comment:
@kyessenov how come we don't just have this embedded in `api_proto_library_internal`? This seems like a bunch of boilerplate that we could potentially eliminate.